src/xcode/ENA/ENA/Source/Extensions/URLSessionConfiguration+Default.swift

```diff
@@ -33,7 +33,21 @@ extension URLSessionConfiguration {
 	}
 
 
-	/// Session config with caching policy `.useProtocolCachePolicy`
+	/// CWA Session config but only valid in WiFi environments
+	/// - Returns: the same as `.coronaWarnSessionConfiguration` but only for WiFi connections
+	class func coronaWarnSessionConfigurationWifiOnly() -> URLSessionConfiguration {
+		let config = coronaWarnSessionConfiguration()
+		// no download over expensive network (cellular)
+		config.allowsExpensiveNetworkAccess = false
+		// no download in case user has activated the low data mode
+		config.allowsConstrainedNetworkAccess = false
+		// session shall not wait for connectivity to become available; fail immediately
+		config.waitsForConnectivity = false
```

Comment:
I think this is not used and can be removed.

Reply:
Even `allowsConstrainedNetworkAccess`?